### DISCUSSION TITLE
Many requests timing out

### DISCUSSION THREAD

**Discussion by blocka** [Budibase/budibase#5460] [Help]
I'm seeing lots of 504's in the network tab, and can't do anything. Edit app, create new app, delete app, etc.

**Comment by gerhardmpl** [comment]
Same here. Working with the builder is not possible (W10Pro, Chrome 100.0.4896.127 or Firefox 99.0.1)

![grafik](https://user-images.githubusercontent.com/40722408/163729514-c7f709dd-c15f-41a0-af6a-588f3fb84342.png)

Edit: opened an issue (#5462) since I believe the service is disrupted. Not sure if a mail to the support account would be better?

**Comment by gerhardmpl** [comment]
@blocka - the Budibase support team restarted some services (see #5462 or #5463), so your issue might be resolved as well.
